req.go

```diff
@@ -581,6 +581,10 @@ func parseCode(id, fileName string, graph reqGraph) error {
 		h.Write([]byte{0})
 	}
 
+	if linepipes.Verbose {
```

Comment:
I don't think we should use linepipes' state to decide the logging level outside of the linepipes itself. Granted, our using a global variable to control the verbosity of linepipes won't win us a medal in software design, and I'd like to only use this variable once in a program.
In this case, I would use go's log.print to write the error message out. Go intentionally doesn't have logging levels, as their idea is that a message is either informative or it's an error, since everybody ignores warnings anyway. I don't fully agree with this reasoning, but there's not much we can do other than using a 3rd party logging library. For now, I think a log.Print would do the trick here (or remove the message).